ompi/mpi/fortran/mpif-h/register_datarep_f.c

```diff
@@ -89,28 +90,28 @@ static int extent_intercept_fn(MPI_Datatype type_c, MPI_Aint *file_extent,
 /* Data structure passed to the intercepts (see below).  It is an OPAL
    list_item_t so that we can clean this memory up during
    MPI_FINALIZE.  */
-typedef struct intercept_extra_state {
+typedef struct ompi_intercept_extra_state {
```

Comment:
I'm pretty sure that one actually came from
    OBJ_CLASS_DECLARATION(intercept_extra_state_t);
and maybe I could have done a more targeted change but I figured all the OBJ_ macros probably work together and would all likely need the same name change. Another option is to fix it at the OBJ_CLASS_DECLARATION macro:

#define OBJ_CLASS_DECLARATION(NAME)             \
    extern opal_class_t NAME ## _class

maybe that should be extern opal_class_t opal_ ## NAME ## _class